frontend/components/FormComponents.tsx

```diff
@@ -32,13 +37,49 @@ interface BasicFormField {
   noLabel?: boolean
 }
 
+/**
+ * This interface allows for completely arbitrary props
+ * to be passed to any of the form inputs.
+ *
+ * The ideal solution would be to explicitly define what
+ * props each form field accepts.
+ *
+ * This is used for now so that we can at least guarantee
+ * that a form field has the basic properties expected
+ * of all form fields.
+ */
 interface AnyHack {
```

Comment:
is there any benefit of `AnyHack` over `any` as a type?

Reply:
I figured that it would be good to specify the purpose of this hack, so that its cleaner to cleanup later. You know which items are applicable to this particular hack and can clean them up easily when the time comes. Either way is okay, I think.

Reply:
Created a task for cleanup. [ch2821]